bool req_0
bool req_1
bool req_2
bool ready
---
bool seat_1_2
bool bridge
bool seat_1_0
bool seat_1_1
bool temp_1
bool temp_0
bool temp_2
bool last_wait_1_0
bool last_wait_1_1
bool last_wait_1_2
bool last_wait_0_1
bool last_wait_0_0
bool last_wait_0_2
bool seat_0_2
bool seat_0_1
bool seat_0_0
bool full
bool wait_0_2
bool wait_0_1
bool wait_0_0
bool wait_1_2
bool wait_1_0
bool wait_1_1
bool loc_2
bool loc_0
bool loc_1
bool temp
bool des